public/app/plugins/panel/heatmap/HeatmapPanel.tsx

```diff
@@ -160,22 +129,11 @@ export const HeatmapPanel = ({
       return null;
     }
 
-    let heatmapType = dataRef.current?.heatmap?.meta?.type;
-    let isSparseHeatmap = heatmapType === DataFrameType.HeatmapCells && !isHeatmapCellsDense(dataRef.current?.heatmap!);
-    let countFieldIdx = !isSparseHeatmap ? 2 : 3;
-    const countField = info.heatmap.fields[countFieldIdx];
-
-    let hoverValue: number | undefined = undefined;
-    // seriesIdx: 1 is heatmap layer; 2 is exemplar layer
-    if (hover && info.heatmap.fields && hover.seriesIdx === 1) {
-      hoverValue = countField.values[hover.dataIdx];
-    }
-
     return (
       <VizLayout.Legend placement="bottom" maxHeight="20%">
         <div className={styles.colorScaleWrapper}>
           <ColorScale
-            hoverValue={hoverValue}
+            // hoverValue={hoverValue}
```

Comment:
do we have a follow-up GH issue to track implementing hover value functionality?

Reply:
Here: https://github.com/grafana/grafana/issues/75879

Reply:
Probably should remove legend changes so that we don't regress current legend behavior